testsuite/unittests/test_neurons_handle_multiplicity.sli

```diff
@@ -47,6 +47,8 @@ M_ERROR setverbosity
              /parrot_neuron
```

Comment:
I think it would be helpful to have one simple example script of how to use the model. The name of the script should be mentioned in the documentation. Maybe a script reproducing one of the figures in the frontiers paper.